### PR TITLE
footer API version: use axios instead of fetchWithAuth

### PR DESCRIPTION
**Pull request by samrichca**
using axios directly for the health API endpoint because it's our only no-auth API request

### CODE REVIEW

File: global/hooks/useHealthAPI.tsx [hunk 1]
```diff
@@ -1,23 +1,21 @@
...
-  const { fetchWithAuth } = useAuthContext();
-
   useEffect(() => {
-    fetchWithAuth({
-      url: API.HEALTH,
-    })
-      .then((res: any) => {
+    axios.get(API.HEALTH, { baseURL: NEXT_PUBLIC_DAC_API_ROOT })
+      .then((res: AxiosResponse | undefined) => {
         setResponse(res);
```
Comment (anncatton) [inline global/hooks/useHealthAPI.tsx:16]: can we check the response is ok before setting the response (and throw error if it is not)?

Comment (samrichca) [reply global/hooks/useHealthAPI.tsx:16]: good idea, will do

Comment (samrichca) [reply global/hooks/useHealthAPI.tsx:20]: added `validateStatus` to axios

Comment (anncatton) [reply global/hooks/useHealthAPI.tsx:20]: oooh fancy

**Review by anncatton — APPROVED** [review]

**Pull request merged**